plugin/fireplace.vim

```diff
@@ -1666,6 +1678,53 @@ augroup fireplace_go_to_file
   autocmd FileType clojure call s:set_up_go_to_file()
 augroup END
 
+" Section: Spec
+
+function! fireplace#qualify_keyword(kw) abort
+  if a:kw =~# '^::.\+/'
+    let kw = ':' . fireplace#resolve_alias(matchstr(a:kw, '^::\zs[^/]\+')) . matchstr(a:kw, '/.*')
+  elseif a:kw =~# '^::'
+    let kw = ':' . fireplace#ns() . '/' . strpart(a:kw, 2)
+  else
+    let kw = a:kw
+  endif
+  return kw
+endfunction
+
+function! s:SpecForm(kw) abort
+  let op = "spec-form"
+  if fireplace#op_available(op)
+    let symbol = fireplace#qualify_keyword(a:kw)
+    let response = fireplace#message({'op': op, 'spec-name': symbol})[0]
+    if !empty(get(response, op))
+      echo s:pr(get(response, op))
+    endif
+  endif
```

Comment:
Drop the conditional and wrap everything atry block to handle the case when we're not connected:

```viml
try
  ...
catch /^Fireplace:/
  return 'echoerr ' . v:exception
entry
```